contracts/src/v0.8/dev/vrf/VRFV2PlusWrapper.sol

```diff
@@ -561,4 +561,17 @@ contract VRFV2PlusWrapper is ConfirmedOwner, TypeAndVersionInterface, VRFConsume
     require(!s_disabled, "wrapper is disabled");
     _;
   }
+
+  /***************************************************************************
+   * Section: Migration of VRFV2PlusWrapper to latest VRFV2PlusCoordinator
+   ***************************************************************************/
+
+  function migrate(address newCoordinator) external onlyOwner {
+    IMigration(address(s_vrfCoordinator)).migrate(SUBSCRIPTION_ID, newCoordinator);
+    s_vrfCoordinator = IVRFCoordinatorV2Plus(newCoordinator);
+  }
+}
+
+interface IMigration {
+  function migrate(uint256 subId, address newCoordinator) external;
 }
```

Comment:
@jinhoonbang do you think we should move this interface definition to a separate sol file?

Reply:
Done. Moved to separate file `IVRFV2PlusMigration`